### PR TITLE
:lipstick: For matter --- get it working

### PR DESCRIPTION
**Pull request by jameshughes89**
### What
Get the formatter working.

### CODE REVIEW

File: src/site/_static/css/custom.css [hunk 1]
```diff
@@ -1,14 +1,13 @@
 /*
     Title and background colors for the class activity.
 */
-.rst-content .activity .admonition-title {
+/*  .rst-content .activity .admonition-title {
```
Comment (twentylemon) [inline src/site/_static/css/custom.css:4]: These seem unrelated.

File: src/test/java/ArrayQueueEqualsTest.java [hunk 2]
```diff
@@ -47,23 +47,6 @@ public void equals_verify_contract() {
...
-                        gManyOneFront,
-                        gManyOneRear)
-                .addEqualityGroup(gManyTwoDefault,
-                        gManyTwoCapacity)
-                .addEqualityGroup(gReverseDefault,
-                        gReverseCapacity)
-                .addEqualityGroup(gTypeDefault,
-                        gTypeCapacity)
-                .testEquals();
+        new EqualsTester().addEqualityGroup(ArrayQueue.class).addEqualityGroup(gEmptyDefault, gEmptyCapacity).addEqualityGroup(gSingletonDefault, gSingletonCapacity).addEqualityGroup(gManyOneDefaultA, gManyOneDefaultB, gManyOneCapacity, gManyOneFront, gManyOneRear).addEqualityGroup(gManyTwoDefault, gManyTwoCapacity).addEqualityGroup(gReverseDefault, gReverseCapacity).addEqualityGroup(gTypeDefault, gTypeCapacity).testEquals();
```
Comment (twentylemon) [inline src/test/java/ArrayQueueEqualsTest.java:50]: Probably want to set it to put new lines in big method chains like this. This is hard to read.

File: src/site/_static/css/custom.css (hunk 1 above)
Comment (jameshughes89) [reply src/site/_static/css/custom.css:4]: i must keep branching from not main. Will become moot once other PR merges.

File: src/test/java/ArrayQueueEqualsTest.java (hunk 2 above)
Comment (jameshughes89) [reply src/test/java/ArrayQueueEqualsTest.java:50]: :eyes:

**Comment by jameshughes89** [comment]
Ok, so, I changed literally nothing this morning and re-ran the formatter and it seems to be doing what is should this time. :shrug:

File: src/site/assignments/maze-solver-too/Assignment3/test/AStarMazeSolverTest.java [hunk 3]
```diff
@@ -14,85 +14,50 @@
...
 
 public class AStarMazeSolverTest {
-    private static final String MAZE_NO_SOLUTION =
-            "#####\n" +
-                    "#   #\n" +
-                    "#   #\n" +
-                    "S  #E\n" +
-                    "#   #\n" +
-                    "#####";
+    private static final String MAZE_NO_SOLUTION = "#####\n" + "#   #\n" + "#   #\n" + "S  #E\n" + "#   #\n" + "#####";
```
Comment (twentylemon) [inline src/site/assignments/maze-solver-too/Assignment3/test/AStarMazeSolverTest.java:17]: You may want to wrap these in:
```java
// @formatter:off
...code...
// @formatter:on
```
The veritcal spacing is intentional here for readability, the formatter can't know that.

You may need to enable the intellij setting for these comments to work: https://www.jetbrains.com/help/idea/reformat-and-rearrange-code.html#exclude_part_of_code

Comment (jameshughes89) [reply src/site/assignments/maze-solver-too/Assignment3/test/AStarMazeSolverTest.java:17]: snazzy
